src/core/helpers/blockies.js

```diff
@@ -0,0 +1,99 @@
+const randseed = new Array(4); // Xorshift: [x, y, z, w] 32 bit values
```

Comment:
This should be replaced by `<mew-blockie />`

Reply:
<mew-blockie /> has printing issue since blockie image is being used as background image.
I'm going to replace this with mew-blockies later when this issue is fixed.
![115953196-e5ac6280-a49e-11eb-8fc9-84bb3f8244eb](https://user-images.githubusercontent.com/9893774/116632763-c1241080-a90c-11eb-810f-d7eef81b6542.png)